src/Uuid/Uuid.php

```diff
@@ -435,6 +429,15 @@ private static function uuid_parse_as_array($uuid)
         );
     }
 
+    private static function toString($v)
```

Comment:
the name is confusing, as it does not always return a string.

Reply:
I get that - naming things... Feel free to submit a PR...